### DISCUSSION TITLE
Launch with all mods disabled?

### DISCUSSION THREAD

**Discussion by stamminator** [ramensoftware/windhawk#334] [General]
Is there a way to launch Windhawk with all mods disabled? I can't find anything in the Wiki or the README about launch parameters.

**Comment by m417z** [comment]
There's no command line flag for it. You can [enable safe mode in the registry](https://ramensoftware.com/windhawk-v1-3-beta#:~:text=Registry%20%E2%80%93%20This%20shouldn%E2%80%99t%20normally%20be%20needed%2C%20but%20safe%20mode%20can%20also%20be%20enabled%20by%20setting%20the%20SafeMode%20value%20to%201%20in%20the%20following%20registry%20key%3A%20HKEY_LOCAL_MACHINE%5CSOFTWARE%5CWindhawk%5CSettings.) before running Windhawk, or if it's already running, enable safe mode via the toolkit (Ctrl+Win+W).
